ci/cpu/build.sh

```diff
@@ -34,10 +35,18 @@ env
 logger "Activate conda env..."
 source activate gdf
 
+logger "Install Openjdk"
+conda install -c anaconda openjdk
+
+logger "Install maven"
+conda install --no-deps -c conda-forge maven
```

Comment:
Don't need to specify the conda channel again, why do we need `--no-deps` here? Would also suggest moving this up to the build environment.